docs/examples/inheritance/07-common-cut.polar

```diff
@@ -23,7 +23,7 @@ exception(actor: Actor, "read", resource: Lab) if
 
 # Not actually sure if this works... needs some testing. If it doesn't work
 # we should think if there is some way to incorporate something like this, because
-# cut() is not ideal if it is common. (Although not sure if the idea of
+# cut is not ideal if it is common. (Although not sure if the idea of
 # exception is much better). Instead we almost want an ALL operator which
 # would change the matching for a given predicate call, ensuring that all
 # match (instead of just one) before returning a true result.
```

Comment:
This entire comment block, from L24-32, looks like it's in need of an update. Doesn't have to be part of this PR.

Reply:
Glancing at it, I would say that the entire file needs an update. It's supposed to be about "Policies where cut is common", but it doesn't have a single `cut`. But we have plenty of bigger fish to fry today.

Reply:
This file isn't used in the documentation yet.